src/cloud-providers/gcp/provider.go

```diff
@@ -117,7 +117,7 @@ func (p *gcpProvider) CreateInstance(ctx context.Context, podName, sandboxID str
 	// "projects/<project>/global/images/<imageid>" or just the "<imageid>" if the
```

Comment:
Please, update this comment to reflect this change.

Reply:
Comment about available options applied